locust/runners.py

```diff
@@ -293,13 +293,13 @@ def shape_worker(self):
         logger.info("Shape worker starting")
         while self.state == STATE_INIT or self.state == STATE_HATCHING or self.state == STATE_RUNNING:
             new_state = self.environment.shape_class.tick()
-            user_count, hatch_rate, stop_test = new_state
-            if stop_test:
+            if new_state == None:
```

Comment:
use `is None` (not that it matters, but it is just nicer :)

Reply:
As you wish ✅